app/src/org/commcare/android/database/user/models/GeocodeCacheModel.java

```diff
@@ -19,8 +19,10 @@
 import java.util.Date;
 
 /**
+ * TODO: Database migration to remove the associated database tables.
  * @author ctsims
  */
+@Deprecated
```

Comment:
This is sweet :clap: 
Android Studio now shows a strikethrough for usages of this class. Gonna use this one more